### ISSUE TITLE
Support for views

### ISSUE THREAD

**Issue by lulco** [lulco/phoenix#248]
ideally structured use like in tables:

```php
$this->view('view_name')->columns(['list of columns'])->select() etc.
```

- [x] create view
- [x] update (replace) view
- [x] drop view
- [ ] rename view
- [ ] dump view
- [ ] tests
- [ ] documentation about views

**Comment by JohnWE** [comment]
Yes, as you say, views are often complex queries that the client wants to run repeatedly.  Lots of joins.  So plain SQL is fine with me.
